charts/trino/templates/configmap-worker.yaml

```diff
@@ -61,6 +61,15 @@ data:
     {{- .Values.server.workerExtraConfig | nindent 4 }}
     {{- end }}
 
+{{- if .Values.accessControl }}{{- if eq .Values.accessControl.type "configmap" }}
```

Comment:
If this duplicates the coordinator part, let's put in `_helpers.tmp`. It's too easy to go out of sync otherwise, we already have another PR open that makes changes here: #208